orchestration/src/main/java/com/sap/ai/sdk/orchestration/spring/OrchestrationSpringChatDelta.java

```diff
@@ -0,0 +1,89 @@
+package com.sap.ai.sdk.orchestration.spring;
+
+import com.google.common.annotations.Beta;
+import com.sap.ai.sdk.orchestration.OrchestrationChatCompletionDelta;
+import com.sap.ai.sdk.orchestration.model.LLMChoice;
+import com.sap.ai.sdk.orchestration.model.LLMModuleResultSynchronous;
+import com.sap.ai.sdk.orchestration.model.TokenUsage;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import lombok.val;
+import org.springframework.ai.chat.messages.AssistantMessage;
+import org.springframework.ai.chat.metadata.ChatGenerationMetadata;
+import org.springframework.ai.chat.metadata.ChatResponseMetadata;
+import org.springframework.ai.chat.metadata.DefaultUsage;
+import org.springframework.ai.chat.model.ChatResponse;
+import org.springframework.ai.chat.model.Generation;
+
+/**
+ * Response from the orchestration service in a Spring AI {@link ChatResponse}.
+ *
+ * @since 1.2.0
+ */
+@Beta
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class OrchestrationSpringChatDelta extends ChatResponse {
+
+  OrchestrationChatCompletionDelta delta;
```

Comment:
(Minor/Question)

Unused field, what's it used for?

Reply:
removed